python/cuml/manifold/t_sne.pyx

```diff
@@ -300,9 +304,31 @@ class TSNE(Base):
         self.post_learning_rate = learning_rate * 2
 
     @generate_docstring(convert_dtype_cast='np.float32')
-    def fit(self, X, convert_dtype=True):
-        """
-        Fit X into an embedded space.
+    def fit(self, X, convert_dtype=True, knn_graph=None):
+        """Fit X into an embedded space.
+
+        Parameters
```

Comment:
I believe this docstring is missing because it gets generated automatically now. Tagging @dantegd for insight

Reply:
That would make sense. I merged the doc strings when I updated with branch-0.16 but have removed them in my most recent commit since they will be auto-generated anyways.

Reply:
It looks like there's still some extra stuff in the docstring. Here's the corresponding example of UMAP's docstring w/ the docstring generator: https://github.com/rapidsai/cuml/blob/3a8fc4ecf0f563f0b46d293f1f3aa9cef838c71a/python/cuml/manifold/umap.pyx#L492

Reply:
I think I was getting confused for no reason here, I've updated the docstrings and they should be good now.